<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EasyModeForm</class>
 <widget class="QWidget" name="EasyModeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1035</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Starting seed</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Opponent</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Number of seeds</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="startSeedBox"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="numbSeedBox"/>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="opponents"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Allowed plays</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="allowedCards">
       <property name="minimumSize">
        <size>
         <width>280</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Number of easy turns</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="easyTurnsBox"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="findEasyBtn">
       <property name="text">
        <string>Find easy modes</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Easy modes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="easyModesBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Enemy plays</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="playsBox">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
